app/poros/github/api.rb

```diff
@@ -22,4 +26,16 @@ def connect
     Faraday.new(url: 'https://api.github.com',
                 params: { access_token: @current_user[:git_hub_token] })
   end
+  
+  def parse_followers
+    parse_info('followers').map { |follower| Follower.new(follower) }
+  end
```

Comment:
Moved the parsing methods into their own helper method, and since none of them were being called outside of the class - we moved them down into a private block.